### PR TITLE
Allow a list of path patterns

### PR DESCRIPTION
**Pull request by jonnyshaw89**
- Allows for multiple paths to be routed to the same container

This PR is intended to allow Author to run on a single url.
This allows for a merge of the author apps to be seamless to the user as the url will not change

### CODE REVIEW

File: ecs.tf [hunk 1]
```diff
@@ -174,7 +175,7 @@ resource "aws_cloudwatch_log_group" "log_group" {
 }
 
 output "service_address" {
-  value = "https://${var.dns_record_name == "" ? aws_route53_record.dns_record.name : var.dns_record_name}"
+  value = "https://${element(coalescelist(aws_route53_record.dns_record.*.name, list(var.dns_record_name)), 0)}"
```
Comment (bitdivision) [inline ecs.tf:128]: Cool, didn't know about this :+1:

File: global_vars.tf [hunk 2]
```diff
@@ -28,7 +28,8 @@ variable "aws_alb_listener_arn" {
 
 variable "alb_listener_path_pattern" {
```
Comment (bitdivision) [inline global_vars.tf]: Variable name should be plural?

File: global_vars.tf [hunk 3]
```diff
@@ -28,7 +28,8 @@ variable "aws_alb_listener_arn" {
...
   description = "The path pattern to match to route to this service"
```
Comment (bitdivision) [inline global_vars.tf]: Update for multiple patterns

**Review by bitdivision — CHANGES_REQUESTED** [review]
Some pedantic points...

**Comment by jonnyshaw89** [comment]
@bitdivision comments addressed

**Review by bitdivision — APPROVED** [review]

**Comment by jonnyshaw89** [comment]
Not really needed yet

**Pull request closed without merging**